### ISSUE TITLE
[MAUI] Bottom Tabs Not Interactable After Suspending Android App

### ISSUE THREAD

**Issue by expensivecow** [roubachof/Sharpnado.Tabs#84]
**Platform (please complete the following information):**
 - OS: [Android]
 - Device: [Android Emulator(Pixel 5)]
 - Sdk Version: [Android SDK API 29]
 - MAUI: [7.0.86]

**Describe the bug**
Hi Roubachof! I am using bottom tabs similar to the provided sample in NET MAUI (https://github.com/roubachof/Sharpnado.Tabs/tree/main/MauiSample). When I suspend the app using the back button and relaunch it, the tabs are no longer interactable. To me, it looks like some states aren't being preserved on app suspension for some reason, but I have not been able to debug the issue further yet. 

Would you have any insights on why this might occur? I'll try to take a look today/tomorrow to see what I can find out otherwise.

**To Reproduce**
1. Launch Sharpnado Tabs Maui Sample (fresh start)
2. Click back button on device to suspend app
3. Re-Launch Sharpnado Tabs Sample

Observed Behavior: Bottom Tabs are not interactable. 

Expected Behavior: Bottom Tabs can be clicked and views load as expected.

**Screenshots (if applicable)**

**Comment by Masgharoth** [comment]
Same issue here. Are there any proper solutions to this or the LaunchMode is just that?

**Comment by Masgharoth** [comment]
Btw: this is not strictly a sharpnado issue at its core. We have a custom control (extending ContentView) that utilizes toucheffects, and it also looses the effect after suspension. Fixed by reregistering the effect in the OnHandlerChanged override. 
Therefore: this is probably a matter of Sharpnado.Tabs handling app suspension/restoration.

**Comment by svaldetero** [comment]
> Fixed by reregistering the effect in the OnHandlerChanged override.

@Masgharoth How'd you go about doing that?

**Comment by janusw** [comment]
@expensivecow @Masgharoth @svaldetero Did anyone manage to work around this issue? If yes, could you please share the exact workaround?

Would be good to finally get this fixed properly ...

**Comment by svaldetero** [comment]
> @svaldetero Did anyone manage to work around this issue? If yes, could you please share the exact workaround?

I did not. My app is running in Kiosk mode, so I just terminated the app when they try to suspend. Not a good approach, but better than the app being unusable after resuming.

```
Platform.CurrentActivity.FinishAffinity();
Environment.Exit(0);
```

**Comment by janusw** [comment]
> > @svaldetero Did anyone manage to work around this issue? If yes, could you please share the exact workaround?
> 
> @janusw I did not. My app is running in Kiosk mode, so I just terminated the app when they try to suspend.

Thanks for the reply! Anyone else? @expensivecow @Masgharoth 

Btw, I can reproduce the problem, as described in the original report (also with .NET 8 on the current main branch).

**Comment by janusw** [comment]
I think I found a fix, see #126. Testing and feedback welcome ...

**Comment by roubachof** [comment]
Guys I'm currently testing it with the MauiSample and I can't reproduce the issue (MAUI 8/Pixel 5), so I guess it was a maui 7 issue
[touch working.webm](https://github.com/user-attachments/assets/395f9aad-7f88-44d2-9063-889767e16dab)

**Comment by roubachof** [comment]
Ok I reproduced it and will be fixed with version 3.1.0

**Comment by roubachof** [comment]
should be fixed by v3.1.0, please re-open if it doesn't.
